flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableTestUtils.java

```diff
@@ -124,4 +128,12 @@ public static void comparedWithKeyAndOrder(
                             matching(TableTestMatchers.deepEqualTo(expectedData.get(key), false)));
         }
     }
+
+    private static String rowToString(Object o) {
+        if (o instanceof Row) {
```

Comment:
This confused me but I understand it is to provide compatibility for 1.19, I'd leave a comment since that's not clear